src/commands/mobile_app/upload.rs

```diff
@@ -249,14 +249,10 @@ fn normalize_directory(path: &Path) -> Result<TempFile> {
         .into_iter()
         .filter_map(Result::ok)
         .filter(|entry| entry.path().is_file())
-        .map(|entry| {
-            let entry_path = entry.into_path();
-            let relative_path = entry_path.strip_prefix(path)?.to_owned();
-            Ok((entry_path, relative_path))
-        })
+        .map(|entry| Ok(entry.into_path()))
         .collect::<Result<Vec<_>>>()?
         .into_iter()
```

Comment:
Wrapping this in an `Ok` is unnecessary, because it is an infallible operation. Also, no need to collect into a vector just to make an iterator again.

```suggestion
        .map(|entry| entry.into_path())
```